law_reader/common/RevisionSummaryInfo.py

```diff
@@ -0,0 +1,13 @@
+class RevisionSummaryInfo:
```

Comment:
So the argument in favor of having a distinct object is that summarize_etl utilizes only certain attributes of Revision rows which are not utilized in other scripts, and that having them be the same object may necessitate cases where the class instance is populated with attributes that are not used in that context (or else having them be set to None in that context). 

Unsure at this moment whether that constitutes sufficient reason for having them be distinct objects, but that is to me the most salient argument.

Reply:
I do not believe there is justifiable reason to have this as a separate object, since it only stores primary keys. If you need that kind of logic, just have it be a tuple with a type alias (RevisionSummaryIds: TypeAlias = Tuple[str, int, int]) that is only used by the summarizer rather than have a new object type that could be used by anyone.

Creating an object is a commitment to reuse the object. And I don't believe this is an object we will find much reuse from.